### ISSUE TITLE
Build Fail

### ISSUE THREAD

**Issue by dyogurt** [nitinagarwal/mesh_slicer#2]
```
[ 90%] Built target nanogui
Scanning dependencies of target mesh_slicer
[ 92%] Building CXX object CMakeFiles/mesh_slicer.dir/main.cpp.o
/home/ubuntu/mesh_slicer/main.cpp:15:10: fatal error: GL/glew.h: No such file or directory
 #include <GL/glew.h>                /* always before GLFW */
               ^~~~~~~~~~~
compilation terminated.
CMakeFiles/mesh_slicer.dir/build.make:62: recipe for target 'CMakeFiles/mesh_slicer.dir/main.cpp.o' failed
make[2]: *** [CMakeFiles/mesh_slicer.dir/main.cpp.o] Error 1
CMakeFiles/Makefile2:67: recipe for target 'CMakeFiles/mesh_slicer.dir/all' failed
make[1]: *** [CMakeFiles/mesh_slicer.dir/all] Error 2
Makefile:83: recipe for target 'all' failed
make: *** [all] Error 2
```
I followed your README instructions but i am getting this fatal error. The error was fixed by installing OpenGL (following instructions [here](https://medium.com/@Plimsky/how-to-install-a-opengl-environment-on-ubuntu-e3918cf5ab6c)). After fixing that, build fails because of missing library `glm` which I also installed. The build is now failing because of `igl` library. Is there a simple way to install all necessary dependencies? The ones listed in the README don't seem sufficient.

**Comment by celstark** [comment]
Having just hit both of these myself...
- For the glm, it's just the typical "apt-get install libglm-dev" route.
- For the igl library, the instructions just said something about "putting it in the path".  What I did was to make a link inside of "mesh_slicer/nanogui/include" to the "libigl/include/igl" folder.

**Comment by yuriao** [comment]
Yes we can copy the "libigl/include/igl" to "mesh_slicer/nanogui/include" to "putting it in the path"
